server/src/OnDeviceComponent.ts

```diff
@@ -379,35 +524,12 @@ export class OnDeviceComponent {
 				e = new Error(message);
 			}
 			throw e;
-		} finally {
-			clearInterval(retryInterval);
-			client?.close();
 		}
 	}
 
 	// Starts up express server
 	private async startServer() {
-		// startServer can be called multiple times. Sometimes before server has even been set so have to wait until ready to proceed
-		if (this.callbackListenPortSetupPromise) {
-			await this.callbackListenPortSetupPromise;
-		}
-
-		if (this.server) {
-			return;
-		}
-
-		let callbackListenPort = this.getConfig()?.callbackListenPort;
-		if (!callbackListenPort) {
-			this.callbackListenPortSetupPromise = portfinder.getPortPromise();
-			callbackListenPort = await this.callbackListenPortSetupPromise;
-		}
-
-		this.debugLog('Starting callback server');
-		this.server = this.app.listen(callbackListenPort, () => {
-			this.debugLog(`Listening for callbacks on ${callbackListenPort}`);
-		});
-		this.callbackListenPort = callbackListenPort;
-
+		// FIXME need to add back in registry restore some how
```

Comment:
Does this need addressed before the PR is complete?

Reply:
Thanks yup I'll go ahead and address this.